src/actions/hwWallets.js

```diff
@@ -0,0 +1,12 @@
+import actionTypes from '../constants/actions';
+
+/**
+ * An action to dispatch a device list to HwWallets
+ *
+ */
+export const devicesListUpdated = data => ({
+  data,
+  type: actionTypes.devicesListUpdate,
```

Comment:
I don't know if this is a typo but looks like the function name is past time a this function action not **devicesListUpdated**.

In the other hand I dont know if should be something like updateDevicesList... what do you think?

Reply:
Update to follow the same standard as we use in other actions and reducers